frontend/src/streamDetails/__tests__/StreamDeviceDetailsCard.test.jsx

```diff
@@ -1,19 +1,17 @@
 import React from "react";
 import Enzyme from "enzyme";
 import Adapter from "enzyme-adapter-react-16";
-import { describe } from "@jest/globals";
+import { afterEach, describe } from "@jest/globals";
 
-import { Grid } from "@material-ui/core";
-import StreamDeviceDetailsCard from "../StreamDeviceDetailsCard";
-import DashboardCard from "../../general/dashboard/DashboardCard";
-import SimpleTable from "../../general/simpleTable/SimpleTable";
+import StreamDetailsDeviceCard from "../StreamDetailsDeviceCard";
+import StreamDeviceCard from "../../general/StreamDeviceCard";
 
 import DeviceInfo from "../../model/DeviceInfo";
 import ButtonInfo from "../../general/dashboard/ButtonInfo";
 
 Enzyme.configure({ adapter: new Adapter() });
 
-describe("<StreamDeviceDetailsCard/> functional component", () => {
+describe("<StreamDetailsDeviceCard/> functional component", () => {
```

Comment:
Rename this file to `StreamDetailsDeviceCard.test.jsx`

Reply:
a2ac648

Reply:
a2ac648